dev/add-test-users.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
```

Comment:
I split this out from the `run-dependencies` script because when running automated tests we don't want any pre-existing permissions (which is how things already run on TC), only for running the app locally to test things manually.